perf/Microsoft.AspNetCore.Server.Kestrel.Performance/configs/DefaultConfig.cs

```diff
@@ -0,0 +1,42 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using BenchmarkDotNet.Configs;
+using BenchmarkDotNet.Engines;
+using BenchmarkDotNet.Environments;
+using BenchmarkDotNet.Jobs;
+using BenchmarkDotNet.Validators;
+
+namespace Microsoft.AspNetCore.Server.Kestrel.Performance
+{
+    public class DefaultConfig : ManualConfig
```

Comment:
Am using the `CoreConfig` for this; will move to default when can move to netcoreapp1.1 https://github.com/dotnet/BenchmarkDotNet/issues/301